project_tests.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 set -e
 
-docker run --rm elife/sciencebeam-gym /bin/bash -c 'venv/bin/pip install pytest nose && venv/bin/pytest sciencebeam_gym'
+pip install -r requirements.dev.txt
```

Comment:
https://github.com/elifesciences/sciencebeam-gym/blob/develop/requirements.dev.txt#L2 still contains `nose`, is that actually used/useful here? `pyest` may cover all test runner concerns.

Reply:
It appears nose was required for Apache Beam test code (i.e. it was just not working, that's something we found a while ago, haven't re-validated with newer versions).